docs/3.1. Discovery - Registered Operation.md

```diff
@@ -68,26 +75,31 @@ The DNS-SD advertisement MUST be accompanied by a TXT record of name 'api\_proto
 
 **api\_ver**
```

Comment:
This needs changing to use `####` like the previous ones, or those should go back to just bold.
I think that other spec documents also use bold rather than headings for TXT records, FWIW.

Reply:
Resolved on the backticks child PR.